src/metadata/misc_parsers.cpp

```diff
@@ -180,7 +180,8 @@ void LibRaw::parse_cine()
   }
   cam_mul[0] = getreal(LIBRAW_EXIFTAG_TYPE_FLOAT);
   cam_mul[2] = getreal(LIBRAW_EXIFTAG_TYPE_FLOAT);
-  maximum = ~((~0u) << LIM(get4(), 1, 31));
+  temp = get4();
+  maximum = ~((~0u) << LIM(temp, 1, 31));
```

Comment:
This fixes the logic error made here: https://github.com/LibRaw/LibRaw/commit/126ce46cd8d6d64a8031eec9164a9d388675ae77#diff-d931b600cedbc8d7b33233b93a215635R183

Adding LIM caused get4() to be called more than once.

Reply:
Agree. It is time to change these macros to inlines (as w/ fMAX), but only after 0.20 release....

Reply:
Strangely enough, this does not affect (very limited set of) cine samples we have.